layouts/partials/scripts.html

```diff
@@ -26,3 +26,13 @@
 {{ $s = $s | append (resources.Get "js/copybutton.js") }}
 {{ $js := $s | resources.Concat "js/bundle.js" | resources.Minify | resources.Fingerprint }}
 <script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}"></script>
+
+<!--
+    These two scripts are provided separately in order to allow browsers to decide which
+    component bundle to load. Modern browsers that support web components will load the
+    smaller bundle. Others will load the (much larger) polyfill. Because they're ultimately
+    responsible for loading child components, whose filenames will contain content-based
+    hashes, they must not be cached.
+-->
+<script type="module" src="/js/components/components.esm.js" defer></script>
```

Comment:
Note that in modern browsers, the payload here will be _very_ small: non-gzipped, the request for `components.esm.js` is 252 bytes, and the subsequent component download (which can be forever-cached) is ~11K.